src/core/getTokenInfo.ts

```diff
@@ -52,6 +58,6 @@ export const getTokenInfo = async (
     return tokenInfo;
   }
 
-  console.error(`Can't get token info from this address: ${tokenAddress}`);
+  toast.error(`Can't get info from tokenAddress: ${tokenAddress}`);
```

Comment:
you can't use `toast` inside of the core function, only in the UI functions/modal